src/domain/users/index.types.d.ts

```diff
@@ -47,7 +48,14 @@ type User = {
 
 interface IUsersRepository {
   findById(userId: UserId): Promise<User | RepositoryError>
+  updateIps(userId: UserId, iPs: IPType[]): Promise<null | RepositoryError>
```

Comment:
Null is not a good return here, please use `true` or `void`

Reply:
this has been deleted